src/commons/sagas/RequestsSaga.ts

```diff
@@ -656,6 +657,7 @@ export const postSourcecast = async (
   const filename = Date.now().toString() + '.wav';
   formData.append('sourcecast[title]', title);
   formData.append('sourcecast[description]', description);
+  formData.append('sourcecast[uid]', uid);
   formData.append('sourcecast[audio]', audio, filename);
   formData.append('sourcecast[playbackData]', JSON.stringify(playbackData));
   const resp = await request(`sourcecast`, 'POST', {
```

Comment:
This is probably late, but why is your component the only one using formdata when everyone else can post json?

Reply:
`multipart/form-data` is needed to pass binary through HTTP.

Reply:
I guess this is more efficient, this is fine (alt: base64 encoding instead of multipart form data). 

Sorry for late reply.